src/Turbo/src/Twig/TwigExtension.php

```diff
@@ -32,6 +38,9 @@ public function getFunctions(): array
     {
         return [
             new TwigFunction('turbo_stream_listen', $this->turboStreamListen(...), ['needs_environment' => true, 'is_safe' => ['html']]),
+            new TwigFunction('turbo_refreshes_with', $this->turboRefreshesWith(...), ['is_safe' => ['html']]),
```

Comment:
I'm not fan of `turbo_refreshes_with` name, I think users will prefer to use `turbo_refresh(method='replace', scroll='reset')... What do you think?

```suggestion
            new TwigFunction('turbo_refresh', $this->turboRefresh(...), ['is_safe' => ['html']]),
```